apps/web/src/components/transactions/TxDetails/Summary/index.tsx

```diff
@@ -73,7 +73,7 @@ const Summary = ({
       )}
 
       {/* Advanced TxData */}
-      {isTxDetailsPreview && (
+      {txData && isTxDetailsPreview && (
```

Comment:
Should we move the entire advanced details into a component and only display it if `txData` is defined? Instead of having the condition on both components?